rpcserver.go

```diff
@@ -6077,6 +6082,27 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 
 			blindingRestrictions.NodeOmissionSet.Add(vertex)
 		}
+
+		blindingRestrictions.IncomingChainedChannels = append(
+			blindingRestrictions.IncomingChainedChannels,
+			blindCfg.IncomingChannelList...,
+		)
+
+		numChainedChannels :=
+			uint8(len(blindingRestrictions.IncomingChainedChannels))
+
+		// When selecting the blinded incoming channel list parameter
+		// the maximum number of hops is implictitly set.
+		if numChainedChannels > blindingRestrictions.NumHops {
```

Comment:
shouldn't rather check:

`numChainedChannels < MinDistanceFromIntroNode` => return error ?

Reply:
correct but I think we need the following check,

if  `MinDistanceFromIntroNode < numChainedChannels` => error out

because imagine selecting one channel and setting the min hops to 0, now you would also include paths which have 0 hops and add it to the invoice, that's what we don't want.

Reply:
the the testcase where you set the MinDistanceFromIntroNode to 0